packages/env/lib/commands/snapshot.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+const fs = require( 'fs' );
+const util = require( 'util' );
+const exec = util.promisify( require( 'child_process' ).exec );
+/**
+ * Internal dependencies
+ */
+const initConfig = require( '../init-config' );
+
+/**
+ * @typedef {import('../wordpress').WPEnvironment} WPEnvironment
+ * @typedef {import('../wordpress').WPEnvironmentSelection} WPEnvironmentSelection
+ */
+
+/**
+ * Takes a snapshot of the development server's database.
+ *
+ * @param {Object}                 options
+ * @param {Object}                 options.spinner     A CLI spinner which indicates progress.
+ * @param {boolean}                options.debug       True if debug mode is enabled.
+ */
+module.exports = async function snapshot( { spinner, debug } ) {
+	const config = await initConfig( { spinner, debug } );
+
+	const snapshotPath = `${ config.configDirectoryPath }/snapshot`;
+	const directoryHash = path.basename( config.workDirectoryPath );
+	const filesVolume = `${ directoryHash }_wordpress`;
+	const dbVolume = `${ directoryHash }_mysql`;
+
+	spinner.text = `Snapshotting ${ directoryHash }.`;
+
+	if ( ! fs.existsSync( snapshotPath ) ) {
+		fs.mkdirSync( snapshotPath );
+	}
+
+	try {
+		await exec(
+			`docker run -v ${ dbVolume }:/volume --rm loomchild/volume-backup backup - > ${ snapshotPath }/db_backup.tar.bz2`
+		);
+		await exec(
+			`docker run -v ${ filesVolume }:/volume --rm loomchild/volume-backup backup - > ${ snapshotPath }/app_backup.tar.bz2`
+		);
+	} catch ( err ) {
```

Comment:
I wonder if it's possible to have the export be a single file instead of two. Might be easier to work with, but a little bit extra code

Reply:
Yea, absolutely.  More portable.  I only kinda know what the script in the docker image being used is doing; likely that technique could be done differently to support multiple volumes.